core-designsystem/src/main/java/com/google/samples/apps/nowinandroid/core/designsystem/component/Chip.kt

```diff
@@ -101,6 +107,80 @@ fun NiaFilterChip(
     )
 }
 
+/**
+ * Now in Android topic chip that displays a topic name text.
+ *
+ * @param enabled Whether the chip is currently enabled.
+ * @param onClick Called when the user clicks the chip for more options.
+ * @param modifier Modifier to be applied to the chip.
+ * @param label The text label content.
+ */
+@Composable
+@OptIn(ExperimentalMaterial3Api::class)
+fun NiaTopicChip(
+    enabled: Boolean,
+    onClick: (Boolean) -> Unit,
+    modifier: Modifier = Modifier,
+    label: @Composable () -> Unit
+) {
+    SuggestionChip(
+        enabled = enabled,
+        onClick = { onClick },
```

Comment:
I think this should just be `onClick = onClick`? Right now, `onClick` won't be called I believe.

Reply:
good catch, fixed